src/cdk-experimental/popover-edit/common.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ReplaySubject} from 'rxjs';
+import {OnDestroy} from '@angular/core';
+
+export const DEFAULT_HOVER_DELAY_MS = 30;
```

Comment:
This is used in two places but it looks like they are mutually exclusive (they don't need to match). If so, can we move this to be two different declarations in each file